src/timeline/TimelineInputs.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+
+class TimelineInputs extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            valid: true
+        };
+        this.handleMinBlur = this.handleMinBlur.bind(this);
+        this.handleMaxBlur = this.handleMaxBlur.bind(this);
+
+    }
+
+    handleMinBlur(e) {
+        console.log(e.target.value);
+        this.props.minYearHandler(e);
+        return null;
+    }
+
+    handleMaxBlur(e) {
+        console.log(e.target.value);
+        this.props.maxYearHandler(e);
+        return null;
+    }
+
+    validateMin(e) {
+        let newValue = e.target.value;
+        console.log(e.target.value);
+        if (newValue > this.props.maxYear) {
+            this.setState({
+                valid: false
+            });
+        } else {
+            this.setState({
+                valid: true
+            });
+        }
+    }
+
+    validateMax(e) {
+        let newValue = e.target.value;
+        console.log(e.target.value);
+        if (newValue < this.props.minYear) {
+            this.setState({
+                valid: false
+            });
+        } else {
+            this.setState({
+                valid: true
+            });
+        }
+    }
+
+    showError() {
+        if (!this.state.valid) {
+            return(
+                <div>
+                    <span className="error">Invalid Year Range</span>
+                </div>
+            )
+        } else {
```

Comment:
you shouldnt' need the "else" here. You should just be able to put "return null" under the "if" and it'll never get there.